### ISSUE TITLE
Landscape mode crashes the iOS app

### ISSUE THREAD

**Issue by adriweb** [nspire-emus/firebird#11]
Backtrace:

```
0	qt_qimageScaleAARGBA_up_xy(QImageScale::QImageScaleInfo *, unsigned int *, int, int, int, int, int, int, int, int)	qimagescale.cpp	316	0x827f06	
1	qt_qimageScaleAARGB(QImageScale::QImageScaleInfo *, unsigned int *, int, int, int, int, int, int, int, int)	qimagescale.cpp	536	0x827c66	
2	qSmoothScaleImage(QImage const&, int, int)	qimagescale.cpp	738	0x827aba	
3	QImage::smoothScaled(int, int) const	qimage.cpp	4275	0x6dfe22	
4	QImage::transformed(QTransform const&, Qt::TransformationMode) const	qimage.cpp	4486	0x6dd54c	
5	QImage::scaled(QSize const&, Qt::AspectRatioMode, Qt::TransformationMode) const	qimage.cpp	2518	0x6dd09e	
6	paintFramebuffer(QPainter *)	qtframebuffer.cpp	50	0x46000	
7	QMLFramebuffer::paint(QPainter *)	qtframebuffer.cpp	65	0x462d8	
8	QSGDefaultPainterNode::paint()	qsgdefaultpainternode.cpp	153	0x4d9064	
9	QSGDefaultPainterNode::update()	qsgdefaultpainternode.cpp	181	0x4d91ae	
10	QQuickPaintedItem::updatePaintNode(QSGNode *, QQuickItem::UpdatePaintNodeData *)	qquickpainteditem.cpp	540	0x57463e	
11	QQuickWindowPrivate::updateDirtyNode(QQuickItem *)	qquickwindow.cpp	2985	0x533570	
12	QQuickWindowPrivate::updateDirtyNodes()	qquickwindow.cpp	2723	0x52ab2c	
13	QQuickWindowPrivate::syncSceneGraph()	qquickwindow.cpp	341	0x52a7b8	
14	QSGRenderThread::sync(bool)	qsgthreadedrenderloop.cpp	512	0x500f3a	
15	QSGRenderThread::syncAndRender()	qsgthreadedrenderloop.cpp	558	0x501402	
16	QSGRenderThread::run()	qsgthreadedrenderloop.cpp	674	0x501f62	
17	QThreadPrivate::start(void *)	qthread_unix.cpp	331	0xc74a2e	
18	_pthread_body	_pthread_body		0x3085de22	
19	_pthread_start	_pthread_start		0x3085dd96	
...	<More>				

```

**Comment by Vogtinator** [comment]
What kind of crash it it? I suspect it's a division by 0.

**Comment by adriweb** [comment]
The painter state looks normal: https://i.imgur.com/oYXSwMI.png

**Comment by Vogtinator** [comment]
Probably fixed in Qt 5.5.1 (which we can't use) and Qt 5.6 (which works, hopefully): https://bugreports.qt.io/browse/QTBUG-48148

**Comment by Vogtinator** [comment]
As Qt 5.5.1 seems to work, this is fixed now and landscape mode will be enabled for the v0.30 release.
